Formula/dgit.rb

```diff
@@ -35,11 +35,13 @@ def install
 
     system "make", "install"
 
-    bin.install "#{ENV["GOPATH"]}/bin/dgit"
+    bin.install "#{ENV["GOPATH"]}/bin/git-dg"
     bin.install "#{ENV["GOPATH"]}/bin/git-remote-dgit"
```

Comment:
I believe this should be `git-remote-dg` now too?

Reply:
Yep, thanks! Should we rename the tap and formula too? `quorumcontrol/decentragit/decentragit`? And then maybe I can figure out how to tell it to switch over on upgrade.